apps/ledger-live-desktop/src/renderer/middlewares/db.ts

```diff
@@ -41,6 +42,10 @@ const DBMiddleware: Middleware<{}, State> = store => next => action => {
     next(action);
     const state = store.getState();
     setKey("app", "postOnboarding", postOnboardingSelector(state));
+  } else if (DB_MIDDLEWARE_ENABLED && action.type.startsWith("MARKET")) {
+    next(action);
+    const state = store.getState();
+    setKey("app", "market", marketStoreSelector(state));
```

Comment:
looking at this approach, it seems that we may have issue on the fact that `MarketState` is currently not serializable because `marketParams: MarketListRequestParams;` isn't because `Date` type isn't, so we may want to select the subset of fields we want to "export" to the store and make the `importMarketState` able to restore a correct state from it.

![Capture d’écran 2024-07-04 à 14 59 15](https://github.com/LedgerHQ/ledger-live/assets/211411/6ada1b03-c0a2-400c-948c-05c3108c2065)

Reply:
It's not used anymore, I'll clean this type